metadoc-tests/src/test/scala/metadoc/tests/MetadocCliTest.scala

```diff
@@ -32,7 +32,7 @@ class MetadocCliTest extends FunSuite {
     // symbol()
     val symbol = out.resolve("symbol")
     assert(Files.exists(symbol))
-    val symbols = symbol.toFile.list()
+    val symbols = symbol.toFile.list().toList
```

Comment:
This looks unnecessary

Reply:
Good catch! Remnants from a debugging session when I wanted to pretty print the array.